src/assets/locales/de/registration.json

```diff
@@ -0,0 +1,47 @@
+{
+  "osp": {
+    "heading": "Registrierungszusammenfassung",
+    "companyHeading": "Firmendetails",
+    "companyMessage": "Unten finden Sie die Registrierungsdaten Ihres Unternehmens, die Ihnen von Ihrem Onboarding-Anbieter mitgeteilt wurden. Bitte überprüfen Sie die Details und fahren Sie mit der Auswahl der Unternehmensrolle unten fort, bevor Sie Ihre Registrierung absenden.",
+    "companyName": "Name der Firma",
+    "bmw": "BMW AG",
+    "street": "Hausnummer",
+    "zip": "Reißverschluss",
+    "city": "Stadt",
+    "region": "Region",
+    "country": "Land",
+    "companyRole": {
+      "title": "Rollenauswahl",
+      "subTitle": "Hinweis: Bitte wählen Sie mindestens eine Rolle aus und akzeptieren Sie die zugehörigen Geschäftsbedingungen.",
+      "TermsAndCondSpan1": "Ja, ich stimme dem zu",
```

Comment:
assuming this should be the same as in English, i needs to be "Ja, ich stimme den"